manager/assets/modext/widgets/security/modx.panel.user.group.js

```diff
@@ -74,7 +70,7 @@ MODx.panel.UserGroup = function(config) {
                                 ,anchor: '100%'
                                 ,listeners: {
                                     'keyup': {scope:this,fn:function(f,e) {
-                                        Ext.getCmp('modx-user-group-header').getEl().update('<h2>'+_('user_group')+': '+Ext.util.Format.htmlEncode(f.getValue())+'</h2>');
+                                        Ext.getCmp('modx-user-group-header').getEl().update(Ext.util.Format.htmlEncode(f.getValue()));
```

Comment:
This is incorrect, should be: `Ext.getCmp("modx-header-breadcrumbs").updateHeader(Ext.util.Format.htmlEncode(f.getValue()));`